lms/product/blackboard/_plugin/course_copy.py

```diff
@@ -0,0 +1,43 @@
+from lms.product.plugin.course_copy import CourseCopyFilesHelper
+from lms.services.file import FileService
+
+
+class BlackboardCourseCopyPlugin:
```

Comment:
Note that if `CourseCopyPlugin` were an `ABC` or a `Protocol` then `BlackboardCourseCopyPlugin` could optionally inherit from it just to explicitly document that it implements that interface (not to inherit any code)